src/renderer/config/global.messages.js

```diff
@@ -0,0 +1,6 @@
+import { defineMessages } from 'react-intl';
```

Comment:
Here it will be like `nodes.messages.js` ? I don't know, should it contain messages for components and app components ?

Reply:
In `global` we can put stuff like "Submit", "Cancel" etc. As for components, I don't know, it was mostly to avoid having 10 times the same translation (e.g. `{count, plural, one {Business Unit} other {Business Units}` in 10 different files.)

Reply:
Yeah sure I see the point but you didn't answer lol